src/control/server/init/setup_spdk.sh

```diff
@@ -17,9 +17,12 @@ else
 	PCI_WHITELIST="$_PCI_WHITELIST" NRHUGE="$_NRHUGE" \
 	TARGET_USER="$_TARGET_USER" "$scriptpath"
 
-	chmod 777 /dev/hugepages
-	chmod 666 /dev/uio*
-	chmod 666 /sys/class/uio/uio*/device/config
-	chmod 666 /sys/class/uio/uio*/device/resource*
+	eet -e
```

Comment:
Typo here?

```suggestion
	set -e
```

But more importantly, this is a duplication of https://github.com/daos-stack/daos/blob/22065fed88ff0e2bf98e8060810a7e2b035c5d1b/src/control/server/init/setup_spdk.sh#L8 isn't it?  My advise would be to just remove this line.